src/frontend/app/features/service-catalog/add-service-instance/select-plan-step/select-plan-step.component.html

```diff
@@ -7,9 +7,13 @@
     </mat-form-field>
   </form>
   <mat-card class="select-plan__details" *ngIf="getSelectedPlan() | async as selPlan">
+    <app-card-status *ngIf="(getPlanAccessibility(selPlan.entity) | async)" [status$]="getPlanAccessibility(selPlan.entity)"></app-card-status>
     <app-metadata-item label="Name"> {{ getDisplayName(selPlan)}} </app-metadata-item>
     <app-metadata-item label="Description"> {{selPlan.entity.entity.description}} </app-metadata-item>
-    <app-metadata-item label="Public"> {{selPlan.entity.entity.public}} </app-metadata-item>
+    <app-metadata-item label="Public">
+      {{selPlan.entity.entity.public}}
```

Comment:
The values of this along with `Free` should be capitalised. Think there's a standard angular filter that will do this.